### PR TITLE
Integration with 3GPP 26.973 v15.1.0 - Test framework

### PR DESCRIPTION
**Pull request by ludomal**
Refers to #85. Depends on #105.

These commits provide the test framework as defined in [3GPP 26.973](https://portal.3gpp.org/desktopmodules/Specifications/SpecificationDetails.aspx?specificationId=3326).
It includes:
- the code for test framework
- test test vectors for the regression test
- the CMake configuration

Furthermore, the excel spreadsheet (Tst_framework.xlsx) contained in the 3GPP archive was converted to a Markdown file.

### CODE REVIEW

**Review by dennisguse — DISMISSED** [review]
Let's merge.

**Comment by ludomal** [comment]
@dennisguse, I have just rebased, updating the commit messages with a [3GPP] tag in front of it.
It looks like it needs approval before merging.

**Review by dennisguse — APPROVED** [review]
Let's go.

**Pull request merged**
